client/src/ui/organisms/footer/index.tsx

```diff
@@ -76,22 +71,22 @@ export function Footer() {
           <ul>
             <li>
               <a
+                className="social-icon twitter"
                 href="https://twitter.com/mozdevnet"
                 target="_blank"
                 rel="noopener noreferrer"
-                aria-label="Twitter"
               >
-                <TwitterIcon />
+                <span className="visuall-hidden">MDN on Twitter</span>
```

Comment:
Typo? `visually-hidden`

Reply:
Yeah, let me change it to Virtually. Btw let me know if other changes are needed as well :)

Reply:
Yup, as Peter mentioned, you can use the existing `visually-hidden` class here and the other places later in the file.